app/views/admin/configs/show.html.erb

```diff
@@ -1006,33 +984,4 @@
       </div>
     </div>
   </div>
-  <% if current_user.has_role?(:single_resource_admin, Config) && current_user.has_role?(:super_admin) %>
```

Comment:
I've been meaning to remove this for a while, as it doesn't really make sense in the Forem world. Just decided to do it here while cleaning things up.